oolab/src/main/java/agh/ics/oop/AbstractWorldMap.java

```diff
@@ -6,11 +6,12 @@ public abstract class AbstractWorldMap implements IWorldMap, IPositionChangeObse
     //protected List<Animal> animals = new ArrayList<>();
     //protected List<IMapElement> elements = new ArrayList<>();
 
-    protected Map<Vector2d, IMapElement> animals = new HashMap<>();
+    protected Map<Vector2d, Animal> animals = new HashMap<>();
+    protected Map<Vector2d, Grass> grasses = new HashMap<>();
 
     @Override
     public boolean place(Animal animal) {
```

Comment:
skoro rzucamy wyjątkiem to nie zwrócimy tu nigdy false -> ta metoda nie powinna już zwracać boolean